torch/nn/_functions/thnn/auto.py

```diff
@@ -164,10 +164,18 @@ def backward(self, grad_output):
                 additional_args = (output,) + additional_args
 
             if is_inplace and self.inplace:
+                inplace_tmp = False 
+                if additional_args[-1]==True:
```

Comment:
Idk. It shouldn't, but I haven't really checked the arguments for all backend functions.